scripts/extract

```diff
@@ -31,7 +31,7 @@ def main(argv):
     with open(filename, 'rb') as fd:
         opts, data = extract(fd)
     for i, (k, v) in enumerate(data):
-        print write_tests(outfile(outdir, module, i), opts, k, v)
+        print (write_tests(outfile(outdir, module, i), opts, k, v))
```

Comment:
I'll take care of it. Thanks.

Reply:
I'm going to `pylint` this file.

Reply:
a897c58

Reply:
Let me know if any of the changes seems to not convey your requirements.

Reply:
How are you running `xref`? `rebar3 xref` shows no missing references.
How are you running `dialyzer`? `rebar3 dialyzer` shows no errors.
(I'm running this through OTP 19 to 23)

Reply:
I was running 23.1 but my setup leaves a lot to be desired, I think it's not configured correctly, will recheck.

Reply:
You just need to `rebar3 xref` it and `rebar3 dialyzer` it.

Reply:
Yes, but in running erlang and rebar3 from their source directory instead of installing it seems to be missing otp libraries, and ERL_LIBS is not doing the trick, so will properly install these. Not a blocker in any way.

Reply:
https://travis-ci.com/github/paulo-ferraz-oliveira/purity/jobs/396071858

`make tests` fails likely because `zsh` is not installed in the image. I can remove that dependency in the future.

Reply:
The units aren't failing; the tests are, but the number of failures increases with OTP version.

Reply:
> Yes, but in running erlang and rebar3 from their source directory instead of installing it seems to be missing otp libraries, and ERL_LIBS is not doing the trick, so will properly install these. Not a blocker in any way.

You can use `kerl` to ease this process.